packages/sanity/src/core/search/common/types.ts

```diff
@@ -133,7 +133,10 @@ export interface TextSearchDocumentTypeConfiguration {
 /**
  * @internal
  */
-export type TextSearchSort = Record<string, {order: SortDirection}>
+export interface TextSearchOrder {
+  attribute: string
+  direction: SortDirection
```

Comment:
Possibly, but `SortDirection` is an existing type used elsewhere in Studio. There is a bit of an impedance mismatch here: Studio typically uses the term "sort", whereas the Text Search API uses "order".

The `TextSearchOrder` type represents data formatted for the Text Search API, with a `direction` field that happens to be compatible with `SortDirection`.

I think if we cared enough to make a change, the best option would be to add a new type alias like this:

```ts
type OrderDirection = SortDirection
```

This is maybe more correct than the current implementation, because the Text Search API is only _accidentally_ compatible with the `SortDirection` type. They could diverge in the future.

What do you reckon? Am I overthinking it? 😅